Tools/autotest/common.py

```
@@ -657,6 +671,21 @@ def run_cmd(self,
                     raise ValueError()
                 break
 
+    def get_default_mode(self):
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_GROUND_ROVER:
+            return "MANUAL"
+        if self.mav.mav_type in [mavutil.mavlink.MAV_TYPE_QUADROTOR,
+                                 mavutil.mavlink.MAV_TYPE_HELICOPTER,
+                                 mavutil.mavlink.MAV_TYPE_HEXAROTOR,
+                                 mavutil.mavlink.MAV_TYPE_OCTOROTOR,
+                                 mavutil.mavlink.MAV_TYPE_COAXIAL,
+                                 mavutil.mavlink.MAV_TYPE_TRICOPTER]:
+            return "STABILIZE"
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_FIXED_WING:
+            return "MANUAL"
+        if self.mav.mav_type == mavutil.mavlink.MAV_TYPE_SUBMARINE:
+            return "STABILIZE"
+
```

Need to raise an exception here.